integration_tests/build_test_containers.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 D3A_IMAGE_TAG="d3a-tests"
-REDIS_IMAGE_NAME="gsyd3a/d3a:redis-staging"
+REDIS_IMAGE_NAME="redis:6.2.5"
 
 if [[ "$(docker images -q ${REDIS_IMAGE_NAME} 2> /dev/null)" == "" ]]; then
     docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}" &&
```

Comment:
You can delete all this, as `docker run` will pull automatically. Thanks!

Reply:
@hannesdiedrich will `docker run` also execute the login?

Reply:
No need to login any more, as redis is publicly available on dockerhub.